src/main/flight/failsafe.c

```diff
@@ -292,8 +294,12 @@ void failsafeUpdateState(void)
                 break;
             case FAILSAFE_GPS_RESCUE:
                 if (receivingRxData) {
-                    failsafeState.phase = FAILSAFE_RX_LOSS_RECOVERED;
-                    reprocessState = true;
+                    if ((failsafeState.validRxDataReceivedAt - failsafeState.validRxDataFailedAt) > (failsafeConfig()->failsafe_recovery_delay * MILLIS_PER_TENTH_SECOND + PERIOD_RXDATA_RECOVERY)) {
```

Comment:
This whole `case` block should be made conditional to USE_GPS_RESCUE.

Reply:
I've moved forward to support the condition in all places in failsafe.c
A little bit of flash space freed for targets not supporting GPS :-)